app/assets/javascripts/request_service.js

```diff
@@ -0,0 +1,34 @@
+var RequestService = function() {
```

Comment:
Didn't consider a singleton - is that better because it doesn't store state so any method can use the same reference and multiple instances aren't required?